src/runtime.js

```diff
@@ -323,20 +348,26 @@ let tornado = {
        return typeof val === 'function';
      },
 
-     /**
-      * Determine Tornado truthiness
-      * @param {*} val The value in question
-      * @return {Boolean}
-      */
-     isTruthy(val) {
-       if (val === 0) {
-         return true;
-       }
-       if (Array.isArray(val) && !val.length) {
-         return false;
-       }
-       return !!val;
-     }
+    /**
+     * Determine Tornado truthiness
+     * @param {*} val The value in question
+     * @return {Boolean}
+     */
+    isTruthy(val) {
+      if (val === 0) {
+        return true;
+      }
+      if (Array.isArray(val) && !val.length) {
+        return false;
+      }
+      return !!val;
+    }
+  },
+
+  attrFilters: {
+    'a[href]': function(val) {
```

Comment:
we may have to repeat this key a bunch of times. a[href], img[src]. is there a way to gather a bunch of contexts to filter on.